src/chunk.c

```diff
@@ -3335,7 +3337,8 @@ find_hypertable_from_table_or_cagg(Cache *hcache, Oid relid)
 						(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 						 errmsg("operation not supported on materialized hypertable"),
 						 errhint("Try the operation on the continuous aggregate instead."),
-						 errdetail("Hypertable \"%s\" is a materialized hypertable.", rel_name)));
+						 errdetail("Hypertable \"%s\" is a materialized hypertable.",
+								   get_rel_name(relid))));
```

Comment:
`rel_name` is already obtained above. I don't understand why it is necessary to obtain it again.